packages/@react-spectrum/button/docs/ActionButton.mdx

```diff
@@ -39,6 +38,7 @@ keywords: [action button]
 
 ```tsx example
 import {Text} from '@react-spectrum/text';
+import Edit from '@spectrum-icons/workflow/Edit';
```

Comment:
@ktabors didn't you say you were having trouble with this at one point? was it resolved then?

Reply:
Seems to be, or maybe a different problem. I didn't see any probs